src/Repository/Mastodon/Account.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Friendica\Repository\Mastodon;
+
+use Friendica\DI;
+use Friendica\Model\APContact;
+use Friendica\Model\Contact;
+use Friendica\Network\HTTPException;
+use Friendica\Repository;
+
+class Account extends Repository
```

Comment:
This one is already a `Factory`, not a `Repository`, because you don't want to "save", "delete" or "update" the created instance anymore. That's the main difference between these two concepts.

=> See the given Friendica Factories, where I excessively use this kind of pattern ;-)